Source/Csla.AspNetCore/Blazor/ApplicationContextManagerBlazor.cs

```diff
@@ -63,21 +63,20 @@ public ApplicationContextManagerBlazor(IHttpContextAccessor hca, AuthenticationS
     /// </summary>
     protected HttpContext HttpContext { get; set; }
 
-    private void InitializeUser()
+    private async Task InitializeUser()
     {
       if (ActiveCircuitState.CircuitExists)
       {
         Task<AuthenticationState> task;
         try
         {
-          task = AuthenticationStateProvider.GetAuthenticationStateAsync();
+          await(task = AuthenticationStateProvider.GetAuthenticationStateAsync());
```

Comment:
This looks kinda odd for me. I would prefer to split it into two lines.
First assign the variable and then await the task.